crates/bevy_window/src/window.rs

```diff
@@ -747,27 +747,27 @@ impl Window {
 /// Defines where window should be placed at on creation.
 #[derive(Debug, Clone, Copy)]
 pub enum WindowPosition {
-    /// Position will be set by the window manager
+    /// The position will be set by the window manager.
     Automatic,
-    /// Window will be centered on the selected monitor
+    /// Window will be centered on the selected monitor.
+    Centered,
+    /// The window's top-left corner will be placed at the specified position (in pixels).
     ///
-    /// Note that this does not account for window decorations.
-    Centered(MonitorSelection),
-    /// The window's top-left corner will be placed at the specified position (in pixels)
-    ///
-    /// (0,0) represents top-left corner of screen space.
+    /// (0,0) represents top-left corner of the selected monitor.
```

Comment:
these two lines feels a lot redundant

Reply:
`window's top-left corner` and `top-left corner of the monitor` are not redundant.